bokehjs/src/coffee/models/sources/geojson_data_source.coffee

```diff
@@ -18,7 +18,7 @@ class GeoJSONDataSource extends ColumnDataSource.Model
   initialize: (options) ->
     super(options)
     @geojson_to_column_data() # this just validates the initial geojson value
-    @register_property('data', @geojson_to_column_data, true)
+    @define_computed_property('data', @geojson_to_column_data, true)
```

Comment:
@bokeh/dev, can someone explain why `GeoJSONDataSource` inherits from `ColumnDataSource`, yet overrides its `data` property with a computed `data` property? @bryevdv, your comment above indicates you were confused about this as well.